### DISCUSSION TITLE
Migrate from local storage to S3

### DISCUSSION THREAD

**Discussion by iamrubeng** [outline/outline#7659] [Self Hosting]
Hello @tommoor,

Is there any procedure to migrate assets from local storage to S3?

**Comment by tommoor** [comment]
I'm not aware of anyone that has gone that direction, but as long as you keep the file structure inside the bucket the same there is no reason it wouldn't work.

**Comment by jberggg** [comment]
@tommoor  I'm wondering the same but the other way around. Is it possible to use the import/export feature or is this only for the page data?

Ah just looked at the documentation which says:

> This import is content only, meaning documents and their attachments, so Workspace settings like Collection permissions, users/groups, etc are not yet imported.

Is there a plan to provide such a feature in the future i.e. that files are included in that export?
